src/pudl/transform/ferc1.py

```diff
@@ -716,6 +716,41 @@ def combine_axis_columns_xbrl(
     return df
 
 
+class AssignQuarterlyDataToYearlyDbf(TransformParams):
+    """Parameters for transfering quarterly reported data to annual columns."""
+
+    quarterly_to_yearly_column_map: dict[str, str] = {}
+    quarterly_filed_years: list[int] = []
```

Comment:
There was a mix of "quarterly-to-yearly" and "yearly-to-quarterly" naming in the classes, functions, and parameters which was confusing. I made it all quarterly-to-early.